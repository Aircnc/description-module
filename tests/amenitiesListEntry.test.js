import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import AmenitiesListEntry from '../client/src/components/amenitiesListEntry.jsx';

describe('AmenitiesListEntry', () => {
  it('should render correctly', () => {
    var output = shallow(
      <AmenitiesListEntry amenity='mockAmenity' />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});