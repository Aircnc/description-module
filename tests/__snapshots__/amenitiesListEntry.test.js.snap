// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AmenitiesListEntry should render correctly 1`] = `
<styled.li>
  mockAmenity
</styled.li>
`;
